### ISSUE TITLE
Accessibility - Landing Page - social icons are missing accessible text

### ISSUE THREAD

**Issue by terrimorton** [dotnet-foundation/website#297]
Page: https://dotnetfoundation.org/

The 4 social icons in the site footer are missing accessible text.

Screenshot showing Accessibility Insights for Web failures:

![image](https://user-images.githubusercontent.com/4643903/89971138-6c2e5600-dc28-11ea-850e-02ab24ff54bc.png)

Full failure details for the Github icon:

**Title**: WCAG 2.4.4,WCAG 4.1.2: Ensures links have discernible text (.fa-github-square)
Tags: Accessibility, WCAG 2.4.4, WCAG 4.1.2, link-name

**Issue**: Ensures links have discernible text (link-name - https://dequeuniversity.com/rules/axe/3.5/link-name?application=msftAI)

**Target application**: .NET Foundation - https://dotnetfoundation.org/

**Element path**: .fa-github-square

**Snippet**: <a class="fab fa-github-square fa-3x" href="https://github.com/dotnet-foundation" target="_blank" alt="GitHub"></a>

**How to fix**: 
Fix all of the following:

-  Element is in tab order and does not have accessible text

Fix any of the following:

-  Element does not have text that is visible to screen readers
-  aria-label attribute does not exist or is empty
-  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
-  Element's default semantics were not overridden with role="presentation"
-  Element's default semantics were not overridden with role="none"

**Environment**: Chrome version 84.0.4147.105

====

This accessibility issue was found using Accessibility Insights for Web 2.20.0 (axe-core 3.5.1), a tool that helps find and fix accessibility issues. Get more information & download this tool at http://aka.ms/AccessibilityInsights.

**Comment by terrimorton** [comment]
An easy way to fix this is to add an aria-label attribute to the anchor tag for each social media icon.

For example: 

> `<a class="fab fa-github-square fa-3x" href="https://github.com/dotnet-foundation" target="_blank" alt="GitHub" aria-label="Find us on GitHub"></a>`

**Comment by Arhell** [comment]
@terrimorton  done

**Comment by terrimorton** [comment]
Thanks, @Arhell! The change works great and has resolved the issue.
